### PR TITLE
electionTimeoutMillis default to 500ms

### PR DESCRIPTION
**Pull request by fracasula**

### CODE REVIEW

File: src/MongoMemoryReplSet.ts [hunk 1]
```diff
@@ -224,7 +224,9 @@ export default class MongoMemoryReplSet extends EventEmitter {
       const rsConfig = {
         _id: this.opts.replSet.name,
         members,
-        settings: this.opts.replSet.configSettings || {},
+        settings: this.opts.replSet.configSettings || {
+          electionTimeoutMillis: 500,
+        },
```
Comment (nodkz) [inline src/MongoMemoryReplSet.ts]: Nope

```js
settings: {
  electionTimeoutMillis: 500,
  ...this.opts.replSet.configSettings
}
```

will be better.

Comment (fracasula) [reply src/MongoMemoryReplSet.ts]: Just realised we had all defaults in the constructor so I moved it there. Looks good?

Comment (nodkz) [reply src/MongoMemoryReplSet.ts]: I think that my variant will be better.

Assume that all works perfectly for some user with defaults settings. And it decides to set other option for RS. And after that, all his test begin to work on 10 seconds longer.

The user must override this `electionTimeoutMillis` only explicitly, setting up other options should not remove this value.

Comment (fracasula) [reply src/MongoMemoryReplSet.ts]: Fixed :+1:

**Comment by nodkz** [comment]
Thanks a lot! 👍
Just saved tons of seconds for testers 👌

**Pull request merged**